### ISSUE TITLE
MySSC: update Drupal to 8.8

### ISSUE THREAD

**Issue by patheard** [dto-btn/todo#55]
The command sequence is:
```bash
# Get latest codebase
git pull
git checkout <branch>

# Check for any new configuration
./drush cex

# Apply codebase changes to environment
./drush updatedb
./drush cim
./drush cr
```

**Comment by patheard** [comment]
Once update, update the Solr config with:
```bash
ansible-playbook play_solr-update.yml -i environments/env --vault-id env@prompt
```

**Comment by patheard** [comment]
Next, add the missing custom blocks:

1. Login as admin.
1. `Structure > Block Layout` from main menu.
1. Remove: "Mega Menu Icon", "Solr Search Block" and "All Rights Reserved" blocks.
1. Create three new custom blocks:
   1. `Custom Block Library` tab.
   1. `Add Custom Block` button.
   1. `Content` block type.
   1. Create the `Mega Menu Icon` and ` Solr Search Footer` blocks, making sure text format is `Full HTML`.

```html
<!-- Mega Menu Icon -->
<a id="menu-toggle-icon" class="toggle mega-menu-toggle-icon" title="Menu" href="#open-menu">
  <span class="icon"></span><span class="label">Menu</span>
</a>


<!-- Solr Search Footer -->
<div class="search-block-form contextual-region block block-search container-inline" data-drupal-selector="search-block-form" role="search">
    <form action="/en/solr/search" method="get" accept-charset="UTF-8">
        <div class="js-form-item form-item js-form-type-search form-type-search js-form-item-keys form-item-keys form-no-label">     
            <label for="edit-keys" class="visually-hidden">What are you looking for today?</label>
            <input title="Enter the terms you wish to search for." data-drupal-selector="edit-keys" type="search" name="search_api_fulltext" value="" size="15" maxlength="128" class="form-search" placeholder="What are you looking for today?">
        </div>
        <div data-drupal-selector="edit-actions" class="form-actions js-form-wrapper form-wrapper">
            <button class="search-form__submit box--purple box--arrow button js-form-submit form-submit" data-twig-suggestion="search_results_submit" data-drupal-selector="edit-submit" type="submit" value="Search">Search</button>
        </div>
    </form>
</div>
```
Next, click `Translate` on the `Solr Search Footer` Edit menu and add the following for the French search:

```html
<!-- Solr Search Footer -->
<div class="search-block-form contextual-region block block-search container-inline" data-drupal-selector="search-block-form" role="search">
    <form action="/fr/solr/search" method="get" accept-charset="UTF-8">
        <div class="js-form-item form-item js-form-type-search form-type-search js-form-item-keys form-item-keys form-no-label">     
            <label for="edit-keys" class="visually-hidden">Qu'est-ce que vous recherchez aujourd'hui?</label>
            <input title="Saisissez les termes que vous souhaitez rechercher." data-drupal-selector="edit-keys" type="search" name="search_api_fulltext" value="" size="15" maxlength="128" class="form-search" placeholder="Qu'est-ce que vous recherchez aujourd'hui?">
        </div>
        <div data-drupal-selector="edit-actions" class="form-actions js-form-wrapper form-wrapper">
            <button class="search-form__submit box--purple box--arrow button js-form-submit form-submit" data-twig-suggestion="search_results_submit" data-drupal-selector="edit-submit" type="submit" value="Chercher">Chercher</button>
        </div>
    </form>
</div>
```

Finally, go back to the `Block Layout` screen and place the blocks:

1. Navigation section: `Place block`.
1. `Mega Menu Icon`, without `Display title`.
1. Move it to the third block in the section.

1. Footer section: `Place block`.
1. `Solr Search Block EN`, without `Display title`.
1. Move it to the third block in the section.

**Comment by patheard** [comment]
Fixed and part of #57
